src/linux/cpu.rs

```diff
@@ -224,7 +224,7 @@ impl CpusWrapper {
 }
 
 /// Struct containing values to compute a CPU usage.
-#[derive(Clone, Copy)]
```

Comment:
Considering this is not user-facing, is there any reason for this change? I don't mind it but just curious what you're hoping to achieve.

Reply:
I was working on debugging cpu frequency feedback in a different peace of hardwares, and for a development point of view it did help having this debug :)

Reply:
Makes sense. Merging then. Thanks!